std/array.d

```diff
@@ -1475,6 +1512,16 @@ unittest //safety, purity, ctfe ...
     assertCTFEable!dg;
 }
 
+///
```

Comment:
Hmm. Not sure whether this belongs here or whether it would be better to just rely on the std.algorithm docs. I guess it's a guaranteed example using arrays, even if the std.algorithm one changes. Debatable.